dotcom-rendering/makefile

```diff
@@ -35,24 +35,10 @@ build: clean-dist install
 	@NODE_ENV=production webpack --config ./scripts/webpack/frontend.js
 
 start: install
-	@make stop
-	$(call log, "starting PROD server...")
-	@echo '' # just a spacer
-	@NODE_ENV=production pm2 start dist/frontend.server.js
-	@echo '' # just a spacer
-	$(call log, "PROD server is running")
-	@pm2 logs
+	@NODE_ENV=production node dist/frontend.server.js
```

Comment:
What happens in this situation now if the node server unexpectedly crashes? Under pm2 it would restart automatically. Presumably the ASG / LB will detect the instance is unhealthy & kill it, but it would be nice to be sure.

Reply:
Don't we start the (actual, live) [server here](https://github.com/guardian/dotcom-rendering/blob/oliver/relative-time/dotcom-rendering/cloudformation.yml#L267)

That's still using pm2

Reply:
In any case, perhaps there's a case for restoring this usage of pm2 locally? I'm happy to do this